sklearn/impute/_base.py

```diff
@@ -256,6 +256,14 @@ class SimpleImputer(_BaseImputer):
      [10.   3.5  9. ]]
     """
 
+    _parameter_constraints = {
+        **_BaseImputer._parameter_constraints,
+        "strategy": [StrOptions({"mean", "median", "most_frequent", "constant"})],
+        "fill_value": ["missing_values"],
```

Comment:
This look weird :)
I am thinking that one could almost pass anything. I don't know if it would be better just to not validate the parameter.

@jeremiedbb any thoughts?

Reply:
I'm curious to know why you think this looks weird. What should we do here?

Reply:
In plain English, we say that the value to be filled is a missing values while the actual goal of the imputer is to replace missing values :)

Reply:
I think "no_validation" is appropriate since any object is valid essentially